app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -857,13 +865,74 @@ fun PostListing(
         PostViewMode.List -> PostListingList(
             postView = postView,
             instantScores = instantScores.value,
+            onUpvoteClick = {
+                instantScores.value = calculateNewInstantScores(
+                    instantScores.value,
+                    voteType = VoteType.Upvote,
+                )
+                onUpvoteClick(it)
+            },
+            onDownvoteClick = {
+                instantScores.value = calculateNewInstantScores(
+                    instantScores.value,
+                    voteType = VoteType.Downvote,
+                )
+                onDownvoteClick(it)
+            },
             onPostClick = onPostClick,
             onPostLinkClick = onPostLinkClick,
             onCommunityClick = onCommunityClick,
             onPersonClick = onPersonClick,
             isModerator = isModerator,
             showCommunityName = showCommunityName,
             account = account,
+            showVotingArrowsInListView = showVotingArrowsInListView,
+        )
+    }
+}
+
+@Composable
+fun PostVotingTile(
+    postView: PostView,
+    instantScores: InstantScores,
+    onUpvoteClick: (postView: PostView) -> Unit,
+    onDownvoteClick: (postView: PostView) -> Unit,
+    account: Account?,
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxHeight()
+            .padding(end = MEDIUM_PADDING),
+    ) {
+        VoteGeneric(
+            myVote = instantScores.myVote,
+            votes = instantScores.upvotes,
+            item = postView,
+            type = VoteType.Upvote,
+            showNumber = false,
+            onVoteClick = onUpvoteClick,
+            account = account,
+        )
+        Text(
+            text = instantScores.score.toString(),
+            style = MaterialTheme.typography.bodyMedium,
+            color = scoreColor(myVote = instantScores.myVote),
+        )
+        // invisible Text below aligns width of PostVotingTiles
+        Text(
+            text = "00000",
```

Comment:
Probably a better way to do this, but I'm fine with it if it works.

Reply:
Agree it feels hacky, but the good thing about this solution is it'll scale properly for the user's font size settings, so won't take up more horizontal room than needed.